lib/driver-helper.js

```diff
@@ -207,7 +207,7 @@ export async function setWhenSettable(
 	driver,
 	selector,
 	value,
-	{ secureValue = false, pauseBetweenKeysMS = 0 } = {}
+	{ secureValue = false, pauseBetweenKeysMS = 1 } = {}
```

Comment:
I think we need to keep this at 0 _only_ for IE

Reply:
@alisterscott Yeah, it looks like that. But [this](https://github.com/Automattic/wp-e2e-tests/blob/master/lib/components/secure-payment-component.js#L35) line confused me, it sets to 1ms if it's IE?